src/components/DefineCompany/BusinessContactInfo.vue

```diff
@@ -152,11 +152,12 @@ export default class BusinessContactInfo extends Vue {
 </script>
 
 <style lang="scss" scoped>
-  [class^="col"] {
-    padding-top: 0;
-    padding-bottom: 0;
-  }
-  .business-contact-container {
+[class^="col"] {
```

Comment:
Sorry, noob question which i've undoubtedly asked before. What exactly does this syntax mean?

Reply:
This selects every element whose class attribute value begins with "col".

https://www.w3schools.com/cssref/css_selectors.asp

Reply:
(^ is start of line in vi; $ is end of line)